include/bluetooth/iso.h

```diff
@@ -114,7 +114,7 @@ struct bt_iso_chan {
 	struct bt_iso_chan_ops		*ops;
 	/** Channel QoS reference */
 	struct bt_iso_chan_qos		*qos;
-	uint8_t				state;
+	enum bt_iso_state		state;
```

Comment:
Is it intentional that you’re increasing the struct member size here. We’ve used packed enums in some places where it matters.

Reply:
IIRC the size of the enum should stay at 8-bit (given the possible values), right? 

In any case, this value is for the host only, and the size of it isn't used for any size-specific fields. 

I can __packed it, but I thought we generally wanted to avoid that in public APIs?

Reply:
AFAIK it’s always promoted to word size if you don’t use packed.

Reply:
Anyway, not an issue if it’s not critical to compact the struct here

Reply:
> AFAIK it’s always promoted to word size if you don’t use packed.

From what I can tell, that's implementation (compiler) specific, and the only requirement is that an enum shall be able hold value _up to_ values representable by an int. 

In any case, this a minor case and I suggest to leave as is. I like the idea of using name enums as fields in the public API, as it gives a direct reference to values that may be used in the field